1-js/02-first-steps/03-strict-mode/article.md

```diff
@@ -1,89 +1,89 @@
 
-````warn header="Ensure that \"use strict\" is at the top"
-Please make sure that `"use strict"` is at the top of your scripts, otherwise strict mode may not be enabled.
+````warn header="Asigurați-vă că \"use strict\" este la început"
+Vă rugăm să vă asigurați că `"use strict"` se află în partea de sus a scripturilor dumneavoastră, în caz contrar, este posibil ca modul strict să nu fie activat.
 
-Strict mode isn't enabled here:
+Modul strict nu este activat aici:
 
 ```js no-strict
 alert("some code");
-// "use strict" below is ignored--it must be at the top
+// "use strict" de mai jos este ignorat - trebuie să fie în partea de sus, la începutul scriptului
 
 "use strict";
 
-// strict mode is not activated
+// modul strict nu este activat
 ```
 
-Only comments may appear above `"use strict"`.
+Doar comentariile pot apărea deasupra `"use strict"`.
 ````
 
-```warn header="There's no way to cancel `use strict`"
-There is no directive like `"no use strict"` that reverts the engine to old behavior.
+```warn header="Nu există nicio modalitate de a anula `use strict`"
+Nu există o comandă de tipul `"no use strict"` care să readucă motorul de căutare ("browserul") la vechiul comportament.
 
-Once we enter strict mode, there's no going back.
+Odată ce am activat modul strict, nu mai există cale de întoarcere.
 ```
 
-## Browser console
+## Consola browserului
 
-When you use a [developer console](info:devtools) to run code, please note that it doesn't `use strict` by default.
+Atunci când folosiți [developer console](info:devtools) pentru a rula codul, vă rugăm să rețineți că aceasta nu folosește `use strict` în mod implicit.
 
-Sometimes, when `use strict` makes a difference, you'll get incorrect results.
+Uneori, când `use strict` face o diferență, veți obține rezultate incorecte.
 
-So, how to actually `use strict` in the console?
+Așadar, cum se utilizează de fapt `use strict` în consolă?
 
-First, you can try to press `key:Shift+Enter` to input multiple lines, and put `use strict` on top, like this:
+În primul rând, puteți încerca să apăsați `key:Shift+Enter` pentru a introduce mai multe linii, și să puneți `use strict` deasupra, astfel:
 
 ```js
-'use strict'; <Shift+Enter for a newline>
-//  ...your code
-<Enter to run>
+'use strict'; <Shift+Enter pentru o linie nouă>
+//  ...codul tău
+<Enter pentru a executa>
 ```
 
-It works in most browsers, namely Firefox and Chrome.
+Funcționează în majoritatea browserelor, mai ales în Firefox și Chrome.
 
-If it doesn't, e.g. in an old browser, there's an ugly, but reliable way to ensure `use strict`. Put it inside this kind of wrapper:
+Dacă nu funcționează, de exemplu, într-un browser vechi, există o modalitate inestetică, dar fiabilă de a asigura `use strict`. Puneți-l în acest tip de înveliș:
 
 ```js
 (function() {
   'use strict';
 
-  // ...your code here...
+  // ...codul tău...
 })()
 ```
 
-## Should we "use strict"?
+## Ar trebui să folosim comanda "use strict"?
```

Comment:
```suggestion
## Ar trebui să folosim "use strict"?
```

Reply:
Înțeleg logica ta, în engleză au folosit un joc de cuvinte ”Should we use strict ?”
Am avut dilema asta, părerea mea este că traducerea este cam ciudată pentru cineva care stie engleză, ”Ar trebui să folosim "use strict"?” 
În capul meu totul se traduce în română și rezută ceva de genul ”Ar trebui să folosim "folosim strict"?” 
Cam asta a fost logica traducerii ”Ar trebui să folosim comanda "use strict"?”